source/AI.cpp

```diff
@@ -3292,20 +3292,40 @@ void AI::MovePlayer(Ship &ship, const PlayerInfo &player, Command &activeCommand
 			if(shift)
 				ship.SetTargetShip(shared_ptr<Ship>());
 
-			double closest = numeric_limits<double>::infinity();
+			double best = numeric_limits<double>::infinity();
 			bool foundEnemy = false;
 			bool foundAnything = false;
+			bool distancePriority = Preferences::Has("Board target");
+
+			auto strategy = [&](const AI &ai) -> function<double(const shared_ptr<Ship> &other)> {
```

Comment:
```suggestion
			auto strategy = [&](const AI &ai) -> function<double(const shared_ptr<Ship> &)> {
```

No argument argument name inside type signatures.